site/docs/tutorials/cdc-join.md

```diff
@@ -0,0 +1,711 @@
+---
+title: Join Change Data Capture Records
+sidebar_label: Join
+description: How to maintain a joined image of Change Data Capture data from multiple tables.
+---
+
+**Change Data Capture** (CDC) refers to the process of **observing
+changes made to a database** and extracting them in a form usable by
+other systems, for the purposes of replication, analysis and many more.
+Basically anything that requires keeping multiple heterogeneous
+datastores in sync.
+
+CDC is especially important to Jet, because it makes possible the
+streaming of changes from databases, in a form efficiently processable
+by Jet. Jet's implementation is based on [Debezium](https://debezium.io/),
+which is an open source distributed platform for change data capture.
+
+In our previous tutorials on how to extract change event data from
+[MySQL](cdc.md) and [Postgres](cdc-postgres.md) databases, we've seen
+how to monitor single database tables and how to maintain up-to-date
+images of those tables in memory, in the form of maps.
+
+Here we will examine how to make our map hold enriched data, combined
+(joined, if you will) from multiple tables. Let's, for example, make
+each of our enriched data objects hold information about one customer
+and all the orders belonging to them.
+
+## 1. Install Docker
+
+This tutorial uses [Docker](https://www.docker.com/) to simplify the
+setup of databases, which you can freely experiment on.
+
+1. Follow Docker's [Get Started](https://www.docker.com/get-started)
+   instructions and install it on your system.
+2. Test that it works:
+   * Run `docker version` to check that you have the latest release
+     installed.
+   * Run `docker run hello-world` to verify that Docker is pulling
+     images and running as expected.
+
+## 2. Start Database
+
+Exact instructions for starting the supported databases are as follows:
+
+* [MySQL](cdc.md#2-start-mysql-database)
+* [Postgres](cdc-postgres.md#2-start-postgresql-database).
+
+## 3. Start Command Line Client
+
+Exact instructions for starting the specific command line client of the
+supported databases are as follows:
+
+* [MySQL](cdc.md#3-start-mysql-command-line-client)
+* [Postgres](cdc-postgres.md#3-start-postgresql-command-line-client)
+
+## 4. Create a New Java Project
```

Comment:
The linked tutorial says to create `cdc-tutorial` project. 

With maven it is fine, there is the artifactId specified.
With gradle it is not defined in the build.gradle file but in settings.gradle and the value defaults to the directory name.

This should be highlighted, because you refer to the jar later. (I created folder called cdc-join and had to change the user code deployment and submit command).